README.md

```diff
@@ -1,15 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-
+ Javascript Capstone 
 > Description the project.
 
+this is a javascript capstone project
+
 
 ## Built With
 
 - Major languages
+
+- HTML, CSS & JS
+- Hint, Stylelint, Eslint
+
 - Frameworks
-- Technologies used
+
+-bootstrap
+-webpack
+
 
 ## Live Demo (if available)
```

Comment:
- [ ] I am unable to find the link to your video presentation on the PR summary or on the README. Kindly ensure you record a video presentation of your project - its length should not be more than 5 minutes.

- [ ] I would recommend you deploy your app and add the link here to the README. If you wouldn't, kindly remove this section from the documentation to keep it neat.

Reply:
thank you